scss/_patterns_notifications.scss

```diff
@@ -35,17 +37,29 @@ $notification-text-margin-bottom: 0.6rem;
       }
     }
 
+    &.is-inset {
```

Comment:
What's `inset`? Do we need another appearance variant? We already have 4.

Reply:
I've taken it out, this was in response to a discussion in the design assembly where @long-chung highlighted the fact that an inset text pattern would be very useful in hour design system for the same reasons it is useful in gov.uk for example. But we can discuss this on a separate issue.